setup.py

```diff
@@ -75,12 +75,18 @@ def run(self):
         'uvicorn',
         'aiohttp',
         'aiodns',
+        'aiofile',
+        'psutil',
         'aiortc>=1.3.1',
         'orjson>=3.6.7',
         'astroplan>=0.8',
         'matplotlib>=3.5.1',
         'uvloop>=0.16',
-        'purepyindi2>=0.0.0',
+        'purepyindi2 @ git+http://git@github.com/xwcl/purepyindi2.git',
```

Comment:
I'm not sure that listing the git URLs here is good. It's better for redistribution, but all of these evolve with MagAO-X software and at least one of them is nontrivial to install (depends on the version of MILK used in the control loop). While this is under heavy development I think all we should do is list them to make sure they exist on the package search path, but not trust pip to install them correctly automatically

Reply:
:) it was an easy way to set up my dev env, but I see your point. I reverted to the non-git requirements list.